portal-kernel/src/com/liferay/expando/kernel/model/ExpandoColumnConstants.java

```diff
@@ -168,6 +174,12 @@
 	public static final String STRING_LOCALIZED_LABEL =
 		"custom.field.java.lang.String.localized";
 
+	public static final String TYPE_DECIMAL = "Decimal";
```

Comment:
We should prefix this with `DATA_` to make it more descriptive and also not to confuse with the actual types.

Rename all three to `DATA_TYPE_DECIMAL`, `DATA_TYPE_INTEGER`, and `DATA_TYPE_TEXT`